### PR TITLE
[risk=no] [DB-947] Use prod data in test/ staging/ stable

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
Use prod data in test/ staging and stable environments

### CODE REVIEW

File: public-api/db-cdr/generate-cdr/move-aggregated-counts.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# This moves aggregated count data csvs from prod bucket to public bucket to be imported to cloudsql
```
Comment (calbach) [inline public-api/db-cdr/generate-cdr/move-aggregated-counts.sh]: Is public being used here to indicate an actual public ACL? Or is this using public in the way that databrowser normally uses public (e.g. as a database name).

In practice, what is the destination project used for --public-project ?

Comment (SrushtiGangireddy) [reply public-api/db-cdr/generate-cdr/move-aggregated-counts.sh]: hmmm, it means public aka "test" bucket/ project.

Comment (calbach) [reply public-api/db-cdr/generate-cdr/move-aggregated-counts.sh]: OK. Should it be called "test-bucket" perhaps? Or something else? I'm just getting confused as to whether this means the ACL on this bucket actually public - is it?

Comment (SrushtiGangireddy) [reply public-api/db-cdr/generate-cdr/move-aggregated-counts.sh]: Ch, changed the wording to be test-bucket. The bucket is not public to all users though. It has access to few service accounts and our team.

File: public-api/db-cdr/generate-cdr/move-aggregated-counts.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# This moves aggregated count data csvs from prod bucket to test bucket to be imported to cloudsql
+
+# End product is:
+# 0) .csv of prod tables in a bucket
+
+# Example usage, you need to provide a bunch of args
+# ./project.rb move-aggregated-counts --prod-project aou-db-prod --public-project aou-db-test \
```
Comment (calbach) [inline public-api/db-cdr/generate-cdr/move-aggregated-counts.sh]: would probably change `--public-project` --> `--test-project` for consistency

**Review by calbach — APPROVED** [review]

**Pull request merged**